### PR TITLE
Add translations to the cloudfront invalidation

### PR DESCRIPTION
**Pull request by steveklabnik**
Sample output:

<details>
```json
{
  "Paths": {
    "Items": [
      "/contribute-tools.html",
      "/faq.html",
      "/google526db64baa692d30.html",
      "/team.html",
      "/CNAME",
      "/README.md",
      "/CONTRIBUTING.md",
      "/links",
      "/contribute-community.html",
      "/user_groups.html",
      "/rust-security-team-key.gpg.ascii",
      "/error.html",
      "/contribute-docs.html",
      "/install.html",
      "/Gemfile",
      "/conduct.html",
      "/rust-key-very-old.gpg.ascii",
      "/rust-key-old.gpg",
      "/LICENSE-MIT",
      "/contribute-compiler.html",
      "/community.html",
      "/friends.html",
      "/contribute-bugs.html",
      "/Gemfile.lock",
      "/favicon.ico",
      "/security.html",
      "/rust-key.gpg.ascii",
      "/ides.html",
      "/documentation.html",
      "/downloads.html",
      "/user-groups.html",
      "/contribute.html",
      "/LICENSE-APACHE",
      "/contribute-libs.html",
      "/legal.html",
      "/index.html",
      "/en-US/contribute-tools.html",
      "/en-US/faq.html",
      "/en-US/team.html",
      "/en-US/contribute-community.html",
      "/en-US/contribute-docs.html",
      "/en-US/install.html",
      "/en-US/conduct.html",
      "/en-US/contribute-compiler.html",
      "/en-US/community.html",
      "/en-US/contribute-bugs.html",
      "/en-US/documentation.html",
      "/en-US/downloads.html",
      "/en-US/user-groups.html",
      "/en-US/contribute.html",
      "/en-US/contribute-libs.html",
      "/en-US/legal.html",
      "/en-US/index.html"
    ],
    "Quantity": 53
  },
  "CallerReference": "travis-Fri-Jul-8-15:10:20-2016"
}
```
</details>

### CODE REVIEW

File: travis/inval-list.py [hunk 1]
```diff
@@ -15,8 +15,16 @@
 def pretty_time():
     return '-'.join(time.strftime('%c').split())
 
+def list_files(site, name):
+    base = os.path.join(site, "en-US")
+    return [os.path.join("en-US", f) for f in os.listdir(base) if os.path.isfile(os.path.join(base, f))]
```
Comment (alexcrichton) [inline travis/inval-list.py]: Should these "en-US" be "name"?

Comment (steveklabnik) [inline travis/inval-list.py]: ah ha, a bad refactoring.

I wanted to try to squeeze line 23 in here too, but it made it too complex

File: payload.json [hunk 2]
```diff
@@ -0,0 +1 @@
...oogle526db64baa692d30.html", "/team.html", "/CNAME", "/README.md", "/CONTRIBUTING.md", "/links", "/contribute-community.html", "/user_groups.html", "/rust-security-team-key.gpg.ascii", "/error.html", "/contribute-docs.html", "/install.html", "/Gemfile", "/conduct.html", "/rust-key-very-old.gpg.ascii", "/rust-key-old.gpg", "/LICENSE-MIT", "/contribute-compiler.html", "/community.html", "/friends.html", "/contribute-bugs.html", "/Gemfile.lock", "/favicon.ico", "/security.html", "/rust-key.gpg.ascii", "/ides.html", "/documentation.html", "/downloads.html", "/user-groups.html", "/contribute.html", "/LICENSE-APACHE", "/contribute-libs.html", "/legal.html", "/index.html", "/en-US/contribute-tools.html", "/en-US/faq.html", "/en-US/team.html", "/en-US/contribute-community.html", "/en-US/contribute-docs.html", "/en-US/install.html", "/en-US/conduct.html", "/en-US/contribute-compiler.html", "/en-US/community.html", "/en-US/contribute-bugs.html", "/en-US/documentation.html", "/en-US/downloads.html", "/en-US/user-groups.html", "/en-US/contribute.html", "/en-US/contribute-libs.html", "/en-US/legal.html", "/en-US/index.html"], "Quantity": 53}, "CallerReference": "travis-Fri-Jul-8-16:17:32-2016"}
```
Comment (alexcrichton) [inline payload.json]: stray file?

Comment (steveklabnik) [inline payload.json]: Gah, it should be in the git ignore. I was too focused on the python...

**Pull request merged**